webpack/helpers.js

```diff
@@ -70,15 +70,15 @@ function getConfig(projectDir, proxyUrl, projectPathConfig, assetsPathConfig, ou
 }
 
 /**
- * Check if user config is added and it is used.s
+ * Check if user config is added and it is used
  *
  * @param {object} config Config object to check.
  * @param {string} key Config object key to test if is false.
  *
  * @since 2.0.0
  */
 function isUsed(config, key) {
-  if (config.hasOwnProperty(key) && config[key]) {
+  if (config.hasOwnProperty(key)) {
```

Comment:
Hmmm but we do need to check if the property is `true/false`, no?

Reply:
closed